internal/pkg/manifest/workload_ecs.go

```diff
@@ -91,15 +91,41 @@ type AlarmArgs struct {
 	MemoryUtilization *float64 `yaml:"memory_utilization"`
 }
 
-// DeploymentConfiguration represents the deployment strategies for a service.
+// WorkerAlarmArgs represents specs of CloudWatch alarms for Worker Service deployment rollbacks.
+type WorkerAlarmArgs struct {
+	AlarmArgs       `yaml:",inline"`
+	MessagesDelayed *int `yaml:"messages_delayed"`
+}
+
+// DeploymentControllerConfig represents deployment strategies for a service.
+type DeploymentControllerConfig struct {
+	Rolling *string `yaml:"rolling"`
+}
+
+// DeploymentConfiguration represents the deployment config for an ECS service.
 type DeploymentConfiguration struct {
```

Comment:
nit: can we rename this to 🥺 to be consistent with `DeploymentControllerConfig` and `WorkerDeploymentConfig`
```suggestion
type DeploymentConfig struct {
```